### ISSUE TITLE
Spreadsheet upload: text edits to clarify pages

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1473]
A number of text edits are proposed:

[wording edits.docx](https://github.com/BiologicalRecordsCentre/iRecord/files/11442063/wording.edits.docx)

**Comment by johnvanbreda** [comment]
Some of the changes have been done, a few others are prepared will not take effect until the code is updated.
A few comments:

The code is simpler if "Currently only showing preferred options." is kept as the last thing in the help text on the import settings controls. So the first example is now:
> Select the survey dataset to import records into.  We recommend using “iRecord Import” for all records, unless you have a good reason not to (e.g. for dragonfly records following BDS guidance). Currently only showing preferred options.

Presumably help on the spatial system codes can be provided in guidance?

For the icon on import summary - "Values in this column are matched to a predefined termlist in iRecord", this text isn't quite right as it's not always a termlist, it can be a looked up activity name, or location name for example. I've changed it to "Values in this column are matched against predefined lists of terms" which is slightly less precise, so the term might be a location name rather than picked from a termlist.

Fixed values that apply to all rows - I actually think this section is useful as it confirms the way the complete record will be built. I've improved the database field name labels though (for next code release).
